pr.ts

```diff
@@ -27,7 +27,7 @@ export const changelog = wrap("Require changelog entries on PRs with code change
   const rootContents: any = await danger.github.api.repos.getContent(getContentParams)
 
   const hasChangelog = rootContents.data.find((file: any) => changelogs.includes(file.name))
-  const markedTrivial = pr.title.match("#trivial")
+  const markedTrivial = [pr.title, pr.body].find(s => s.includes("#trivial"))
```

Comment:
No biggie, but you could avoid the array (if you want) by doing: 

`(pr.title + pr.body).include?("#trivial")` 

(I actually stole this from danger.systems IIRC 😝 )